.travis.yml

```diff
@@ -45,37 +40,46 @@ jobs:
 
     - stage: "Build the backend and upload a Docker image"
       before_script:
+        - sudo apt-get install -y libgmp-dev
         - curl -sSL https://get.haskellstack.org/ | sh
+        - stack setup
```

Comment:
Installs GHC (same as before but it's nicer to do it as a separate step and get Travis timings for this step)